test/src/main/java/org/springframework/security/test/web/servlet/request/SecurityMockMvcRequestBuilders.java

```diff
@@ -260,6 +298,25 @@ public FormLoginRequestBuilder acceptMediaType(MediaType acceptMediaType) {
 
 		private FormLoginRequestBuilder() {
 		}
+
+
+		@Override
+		public boolean isMergeEnabled() {
+			return true;
+		}
+
+		@Override
+		public Object merge( Object parent ) {
```

Comment:
Remove extra space for consistent formatting

```suggestion
		public Object merge(Object parent ) {
```